### PR TITLE
Restructure code for new release

### LINKED ISSUE #147: Opening InputStream Helper info pane hangs Kodi
When you open the InputStream Helper add-on information pane (either by clicking the add-on in the "Add-ons / Recently updated" menu, or by following the dependency of a package), your Kodi completely freezes.

This happened to me on LibreELEC v9.0.2 (Raspberry Pi) as well as on Windows 10 Kodi v18.4.

### PR DESCRIPTION
**Pull request by dagwieers**
This restructures the code:
- Rename addon.py into ishelper.py
- Move api.py and config.py out of the global module namespace
- Fix tests to accommodate this

This fixes #147 
This relates to #94

### CODE REVIEW

**Comment by dagwieers** [comment]
So while we did not want to reorganize the code already, the fact we are now exposing `api.py` in the module namespace would make think we have to do it now.

The main code is now in `__init__.py` for backward compatibility, and we probably want to split it up in the future too, but this requires a design decision that I don't want to have now. We probably want to think how we would like developers to use inputstreamhelper in the future.

**Comment by dagwieers** [comment]
@emilsvennesson Did we disable codecov reporting completely at the org level?
I don't understand why there is no codecov check anymore.

**Comment by emilsvennesson** [comment]
@dagwieers I haven't touched anything related to codecov. I guess we need to recheck the settings again.

**Comment by mediaminister** [comment]
No problems found on Windows, Linux, LibreELEC, Android

File: default.py [hunk 1]
```diff
@@ -2,7 +2,9 @@
 ''' This is the actual InputStream Helper API script entry point '''
 
 from __future__ import absolute_import, division, unicode_literals
+import os
 import sys
-from api import run
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'lib'))  # noqa: E402
```
Comment (mediaminister) [inline default.py]: Will `os.path.realpath(__file__)` always work on all platforms? Maybe using the xbmc Python api for this is safer:
```python
from kodi_six import xbmc, xbmcaddon
sys.path.append(os.path.join(xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path')), 'lib'))
```

Comment (dagwieers) [reply default.py]: Yes, using os.path.realpath(__file__) is the best way to do it.

Comment (mediaminister) [reply default.py]: Okay, but I hope we don't  break the rules using `os.path.realpath(__file__)`
https://kodi.wiki/view/Add-on_rules#Requirements_for_scripts_and_plugins

Comment (dagwieers) [reply default.py]: I see other add-ons doing this too, but you can never be sure.

**Pull request merged**